src/services/UtilService.ts

```diff
@@ -40,10 +41,67 @@ const fetchShipmentMethodTypes = async (payload: any): Promise <any> => {
   });
 }
 
+const askQuery = async (payload: any): Promise <any> => {
+  const token = process.env.VUE_APP_GITBOOK_API_KEY;
+  const baseURL = `https://api.gitbook.com/v1/spaces/${process.env.VUE_APP_SPACE_ID}/search/`
+
+  return await client({
+    url: `ask`, 
+    method: "post",
+    baseURL,
+    data: {
+      "query": payload.queryString
+    },
+    headers: {
+      Authorization:  'Bearer ' + token,
+      "Content-Type": "application/json",
+      "Access-Control-Allow-Origin":   "*"
+    }
+  }) as any;
+}
+
+const getGitboookPage = async (pageId: any): Promise <any> => {
+  const token = process.env.VUE_APP_GITBOOK_API_KEY;
+  const baseURL = `https://api.gitbook.com/v1/spaces/${process.env.VUE_APP_SPACE_ID}/`
```

Comment:
We can keep the gitbook api endpoint in the env `https://api.gitbook.com/v1/spaces/`

```suggestion
  const baseURL = `https://api.gitbook.com/v1/spaces/${process.env.VUE_APP_SPACE_ID}/`
```